vnode_controller/vnode_controller.go

```
@@ -489,50 +405,51 @@
-		WorkerNum:         brc.vNodeWorkerNum,
-	}, t)
+		WorkerNum:         vNodeController.vNodeWorkerNum,
+	}, vNodeController.tunnel)
 	if err != nil {
 		err = errpkg.Wrap(err, "Error creating vnode")
 		return
 	}
 
 	// Store the VNode in the runtime info store
-	brc.runtimeInfoStore.PutVNode(nodeID, vn)
+	vNodeController.vNodeStore.AddVNode(nodeName, vn)
```

golangci-lint

Error return value of `vNodeController.vNodeStore.AddVNode` is not checked

(errcheck)